docs/preset-env.md

```diff
@@ -302,7 +328,38 @@ var b = new Map();
 
 #### `useBuiltIns: false`
 
-Don't add polyfills automatically per file, or transform `import "@babel/polyfill"` to individual polyfills.
+Don't add polyfills automatically per file, or transform `import "core-js"` to individual polyfills.
+
+### `corejs`
+
+`CorejsVersion` or `{ version: CorejsVersion, proposals: boolean }`, defaults to `2`.
+`CorejsVersion` can be either `2` or `3`.
+
+This option, which can only be used when `useBuiltIns` is not `false`, configures `@babel/preset-env` to inject imports to the correct `core-js` version.
+By default, only polyfills for stable ECMAScript features are injected: if you want to polyfill them, you need to set `proposals: true`.
```

Comment:
We should explain why `proposals: true` is necessary and is different than `useShippedProposals` (which I'm still fuzzy on, personally).